packages/vue-app/types/index.d.ts

```diff
@@ -63,3 +63,12 @@ export interface LoadingObject {
   start(): void;
```

Comment:
These typedefs are no longer needed, they have been moved directly in `NuxtApp` interface 😉

Reply:
https://github.com/nuxt/nuxt.js/blob/78580a6fa73f065c775f0dbb09a27c7d6e2ec201/packages/vue-app/types/index.d.ts#L62